### PR TITLE
Added initial settings page for the users (Reference used : Facebook settings page) 

### LINKED ISSUE #335: Create a setting page for a user
**Present scenario**
As of now, there is no setting page, 

**Deliverables:** 
so create a setting page where users will have options similar to the Facebook settings page (for reference).

**Working on it.**

### PR DESCRIPTION
**Pull request by Rupeshiya**
Added initial settings page for the users (Reference used: Facebook settings page) 

**Screenshot**
settingsPage

No hardcoded data is used(except default data), keeping the components open for receiving data from API.

Fixes #335 
@vaibhavdaren @jaskirat2000 Please review

### CODE REVIEW

**Comment by jaskirat2000** [comment]
Please remove donut logo from right side

**Comment by Rupeshiya** [comment]
@jaskirat2000 Done!
desk

**Comment by Rupeshiya** [comment]
@jaskirat2000  Update the code with popups.
Only left with sidebars works will do it now.
@vaibhavdaren, please review




React App

**Comment by devesh-verma** [comment]
@Rupeshiya can you please squash the commits

**Comment by Rupeshiya** [comment]
Done! @devesh-verma Please review now.

File: src/common/Popups.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+}
+
+Popups.propTypes = {
+  modalShow: PropTypes.bool.isRequired,
+  option: PropTypes.string.isRequired,
+  optionValue: PropTypes.any.isRequired
+}
+
+
+export default Popups;
```
Comment (vaibhavdaren) [inline src/common/Popups.js]: new line at eof

File: src/user/dashboard/settings/styles/settings.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.settings {
+  display: flex;
+  min-height: 100vh;
+  height: auto;
+  font-family: Muli, sans-serif;
+
+  .navigation {
+    flex: 0.5;
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply src/user/dashboard/settings/styles/settings.scss]: Can you please let me know what's the issue here?

File: src/user/dashboard/settings/layouts/SettingSidebar.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import "../../../dashboard/navigation/navigation.scss";
+import { DonutTitleSmall } from '../../../../donutTitle/donutTitle';
+
+class SettingSidebar extends Component {
+  render() {
+    const iconsSize = 40;
+    return (
+     <div className="navigation">
+        <ListGroup>
+          <ListGroup.Item className={this.props.privacy ? "active mt-4" : "inactive mt-4"} >
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply src/user/dashboard/settings/layouts/SettingSidebar.js]: @vaibhavdaren I am unable to get the issue.
Can you please let me know what's the issue here?

**Pull request merged**